doc/services/portability/posix/option_groups/index.rst

```diff
@@ -603,18 +603,21 @@ Additional POSIX Options
 _POSIX_ASYNCHRONOUS_IO
 ++++++++++++++++++++++
 
+Functions part of the ``_POSIX_ASYNCHRONOUS_IO`` Option are not supported in Zephyr at this time.
+These functions will fail with ``ENOSYS`` :ref:`†<posix_undefined_behaviour>`.
+
```

Comment:
ditto what I raised in #73964. Could you please find a better wording. Another suggestion, including a ".. warning" to make things stand out more.


```suggestion
.. warning::

   While implemented, the functions part of the ``_POSIX_ASYNCHRONOUS_IO`` Option are not supported
   in Zephyr at this time.
   These functions will fail with ``ENOSYS`` :ref:`†<posix_undefined_behaviour>`.

```

Reply:
Fixed! (I hope)